lib/fastlane/actions/crashlytics.rb

```diff
@@ -35,7 +35,7 @@ def self.run(params)
           Helper.log.info 'Build successfully uploaded to Crashlytics'.green
         else
           Helper.log.fatal 'Error uploading to Crashlytics.'
-          raise 'Error when trying to upload ipa to Crashlytics'.red
+          fail 'Error when trying to upload ipa to Crashlytics'.red
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.